fuse/post_officer_history.py

```diff
@@ -34,15 +34,16 @@ def fuse_events(pprr):
 
 
 if __name__ == "__main__":
-    post = pd.read_csv(deba.data("match/post_officer_history.csv"))
+    post = pd.read_csv(deba.data("clean/post_officer_history.csv"))
```

Comment:
Any reason for discarding the match result?

Reply:
I used the wrong decision score for match. Fixed.